package.json

```diff
@@ -19,10 +19,10 @@
     "build:ts": "wireit",
     "build:lit-starter-ts": "wireit",
     "format": "npm run format:eslint && npm run format:prettier",
-    "format:eslint": "eslint \"**/*.{js,ts}\" --fix",
+    "format:eslint": "eslint \"packages/**/*.{js,ts}\" --fix",
```

Comment:
This is a change in that linting is currently handling top-level config files and files in scripts/.

I would add scripts and file an issue to mode rollup-common.js into there or some other config folder. We also have packages/internal-scripts

Reply:
I reverted back to the same glob `**/*.{js,ts}` and instead added option `--ignore-pattern=examples/` to the command so this'll still work the same as before for top level config files and scripts/. WDYT?

Reply:
sounds great!